featuretools/entityset/entityset.py

```diff
@@ -819,7 +819,7 @@ def add_last_time_indexes(self, updated_entities=None):
                         lti.index = entity.df[entity.index].copy()
                         lti = lti.apply(lambda x: None)
                     elif is_instance(entity.df, ks, 'DataFrame'):
-                        lti = ks.Series(index=lti.to_list()).rename(lti.name)
```

Comment:
I see we're removing the rename section of this line - is it no longer necessary?

Reply:
Good question - nothing in the tests seem to rely on the series being named, so either it isn't necessary or we aren't covering a situation in our tests where it is...

Reply:
instead of the rename call, can we use the `name` parameter of the ks.Series constructor?

Reply:
Had to use it on the pd.Series instead - ks.Series won't accept a name if the data is a pd.Series, but it looks like it takes on the name of the pd.Series.